server/http.go

```diff
@@ -302,15 +445,89 @@ func (p *Plugin) handleMeetingEnded(w http.ResponseWriter, r *http.Request, webh
 
 	w.Header().Set("Content-Type", "application/json")
 	if err := json.NewEncoder(w).Encode(post); err != nil {
-		p.API.LogWarn("failed to write response", "error", err.Error())
+		http.Error(w, "failed to write the response", http.StatusInternalServerError)
 	}
 }
+func (p *Plugin) slackAttachmentToUpdatePMI(currentValue string, channelID string) model.SlackAttachment {
+	apiEndPoint := fmt.Sprintf(APIToUpdatePMI, manifest.ID)
+
+	slackAttachment := model.SlackAttachment{
+		Fallback: "You can not set your preference",
+		Title:    "*Setting: Use your Personal Meeting ID*",
+		Text:     fmt.Sprintf("\n\nDo you want to use your Personal Meeting ID when starting a meeting?\n\nCurrent value: %s", currentValue),
+		Actions: []*model.PostAction{
+			{
+				Id:    Yes,
+				Name:  Yes,
+				Type:  "button",
```

Comment:
```suggestion
				Type:  model.POST_ACTION_TYPE_BUTTON,
```
Do this everywhere where needed.

Reply:
`POST_ACTION_TYPE_BUTTON` is not declared by package model

Reply:
Use `PostActionTypeButton` then.